source/test/E2E.test.js

```diff
@@ -1,82 +1,111 @@
 
 
-    
-
-
-
-test('Check thumnails on home page',async () => {
-    const puppeteer = require("puppeteer");
-    const browser = await puppeteer.launch();
-    const page = await browser.newPage();
-    await page.goto("https://big-bytez.github.io/cse110-fa21-group19/source/home-page/",{
-        waitUntil: 'networkidle0',
-      });
-    
-    //wait until the thumnails are generated
-    const numContainer = await page.$$eval(".container", (prodItems) => {
-        return prodItems.length;
-    });
-    const numThumbnails = await page.$$eval('search-recipe', (prodItems) => {
-        return prodItems.length;
-    });
-
-
-    await browser.close();
-
-    expect(numThumbnails).toBe(15);
-
-},10000);
-
-    // it("Check recipes page routing", async () => {
+    test('Check thumnails on home page',async () => {
+        const puppeteer = require("puppeteer");
+        const browser = await puppeteer.launch();
+        const page = await browser.newPage();
+
+        // goto main page 
+        await page.goto("https://big-bytez.github.io/cse110-fa21-group19/source/home-page/index.html",{
+            waitUntil: 'networkidle0',//wait until all source loaded
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8454745)
```suggestion
            waitUntil: "networkidle0",//wait until all source loaded
```